src/Algorithms/BackwardEuler.jl

```diff
@@ -0,0 +1,53 @@
+"""
+$(TYPEDEF)
+
+Backward Euler's integration scheme with given step-size.
+
+### Fields
+
+- `Δt` -- step-size
+
+"""
+struct BackwardEuler{N} <: AbstractSolver
+    Δt::N
+end
+
+BackwardEuler(; Δt::T) where N = BackwardEuler(Δt)
+
+function _init(alg::BackwardEuler, M, C, K)
+    Δt = alg.Δt
+    K̂ = K * Δt + C
+    return K̂
+end
+
+function _solve(alg::BackwardEuler{N},
+                ivp::InitialValueProblem{<:SecondOrderAffineContinuousSystem{N}, XT},
+                NSTEPS::Int) where {N, VT, XT<:Tuple{VT, VT}}
+
+    sys = system(ivp)
+    (U₀) = initial_state(ivp)
+
+    IMAX = NSTEPS + 1
+    M, C, K, R = _unwrap(sys, IMAX)
```

Comment:
after this line we can add this check

````julia
iszero(M) || throw(ArgumentError("this method assumes that the system is of first order"))
```

Reply:
done in commit https://github.com/ONSAS/StructuralDynamicsODESolvers.jl/pull/7/commits/318910365815ec1271979431c907bcafd2cf0810